cosmoz-bottom-bar.js

```diff
@@ -402,6 +403,14 @@ class CosmozBottomBar extends PolymerElement {
 			this.style.maxHeight = to;
 		});
 	}
+
+	_listenForButtonClicks() {
+		window.addEventListener('keydown', (event) => {
+			if (event.shiftKey && event.key === 'A') {
+				window.dispatchEvent(new CustomEvent('toggle-action-list'));
+			}
+		});
+	}
```

Comment:
I don't think that this code needs to be here. First of all the name `toggle-action-list` does not make sense in the context of this component. What action list?
Second: all it does is dispatch an event when Shift+A is pressed. As the code stands, there is no reason for this round-aboutness. You can easily move the code to the app and skip the need for a `toggle-action-list` event. Maybe I am missing something. How does this help you?